src/app/containers/MostRead/Canonical/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useEffect, useState } from 'react';
+import { string } from 'prop-types';
+import nodeLogger from '#lib/logger.node';
```

Comment:
Since we'll be running this on the client side, it'd be good to use  the web logger instead of the node logger. 
```
import webLogger from '#lib/logger.web';

const logger = webLogger();
```